keras_nlp/models/gpt2/gpt2_preprocessor.py

```diff
@@ -63,68 +62,63 @@ class GPT2Preprocessor(Preprocessor):
         add_end_token: If true, the preprocessor will append the tokenizer
             end token to each input sequence.
 
+    Call arguments:
+        x: A string `tf.Tensor` or list of python strings.
+        y: Any label data. Will be passed through unaltered.
+        sample_weight: Any label weight data. Will be passed through unaltered.
+        sequence_length: Pass to override the configured `sequence_length` of
+            the layer.
+        add_start_token: Pass to override `add_start_token` on the layer.
```

Comment:
Seems here `add_start_token` should be set as a string, while in `__init__` it is a bool. Shall we avoid saying "override" here?

Reply:
I'm confused at this comment, `add_start_token` should never be a string. What do you mean?